src/ode_solver/pod_petrov_galerkin_ode_solver.h

```diff
@@ -2,10 +2,17 @@
 #define __POD_PETROV_GALERKIN_ODE_SOLVER__
 
 #include "dg/dg.h"
-#include "implicit_ode_solver.h"
+#include "ode_solver_base.h"
 #include "linear_solver/linear_solver.h"
-#include "reduced_order/pod_basis.h"
+#include "reduced_order/pod_interface.h"
 #include <deal.II/lac/trilinos_sparsity_pattern.h>
+#include <EpetraExt_MatrixMatrix.h>
+#include <Epetra_Vector.h>
+#include <deal.II/lac/la_parallel_vector.h>
+#include <Epetra_Vector.h>
+#include <Epetra_LinearProblem.h>
+#include "Amesos.h"
+#include "Amesos_BaseSolver.h"
 
```

Comment:
same comment as i left for the includes; try to apply this throughout the entire PR